scripts/setup-passcode.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [[ -z $OKTA_API_KEY ]]; then
+    echo "No Org API key"
+    exit 1
+fi
+
+if [[ -z $OKTA_DOMAIN ]]; then
+    echo "No Domain"
+    exit 1
+fi
+
+oktamate=./scripts/oktamate
+
+# Delete user if it exists to not fail the script
+existing_user=$($oktamate users -k $OKTA_API_KEY -d $OKTA_DOMAIN | grep $PASSCODE_USERNAME)
+if [[ -n $existing_user ]]; then
+    $oktamate users delete -k $OKTA_API_KEY -d $OKTA_DOMAIN $PASSCODE_USERNAME
+fi
+
+$oktamate users create -k $OKTA_API_KEY -d $OKTA_DOMAIN -p $PASSCODE_PASSWORD -g '00gua62f9B5eLhVBx5d6' $PASSCODE_USERNAME
```

Comment:
The ID will be unique for each org; you should use the group name here, instead.

Reply:
Done